pkgs/applications/video/olive-editor/default.nix

```diff
@@ -1,34 +1,68 @@
-{ lib, stdenv, fetchFromGitHub
-, pkg-config, which, qmake, wrapQtAppsHook
-, qtmultimedia, frei0r, opencolorio_1, ffmpeg-full, CoreFoundation }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  which,
+  wrapQtAppsHook,
+  frei0r,
+  opencolorio,
+  ffmpeg-full,
+  CoreFoundation,
+  cmake,
+  openimageio2,
```

Comment:
`nix-review` is unhappy about this one:

```
error: Function called without required argument "openimageio2" at /home/aengelen/.cache/nixpkgs-review/pr-222710-1/nixpkgs/pkgs/applications/video/olive-editor/default.nix:13, did you mean "openimageio" or "openimageio_1"?
```
Though I'm not sure why, as `openimageio2` is in `pkgs/top-level/aliases.nix`